### ISSUE TITLE
test "erf table" failing

### ISSUE THREAD

**Issue by vcunat** [haskell/math-functions#63]
```
      erf table:                                             FAIL
        tests/Tests/SpecFunctions.hs:444:
         x         = -4.0e-2
         expected  = -4.511110614512475e-2
         got       = -4.511110614512474e-2
         ulps diff = 2
         err.est.  = 1
```

It's math-functions-0.3.4.0 + ghc-8.8.3 + x86_64 macOS + [nix packager](https://nixos.org/nix).

**Comment by Shimuuar** [comment]
erf normally just wraps function from libc. I guess macos uses different libc.

**Comment by vcunat** [comment]
Yes, they certainly do :-)

**Comment by Shimuuar** [comment]
Should be fixed in 0.3.4.3
